scripts/kafka-integration-test.sh

```diff
@@ -3,9 +3,49 @@
 set -e
 
 export STORAGE=kafka
-while true; do
+
+# Function to start Kafka
+start_kafka() {
+    echo "Starting Kafka..."
+    
+    docker run --name kafka \
+    --network jaeger \
+    -p 9092:9092 \
+    -e KAFKA_CFG_NODE_ID=0 \
+    -e KAFKA_CFG_PROCESS_ROLES=controller,broker \
+    -e KAFKA_CFG_CONTROLLER_QUORUM_VOTERS=0@kafka:9093 \
+    -e KAFKA_CFG_LISTENERS=PLAINTEXT://:9092,CONTROLLER://:9093 \
+    -e KAFKA_CFG_ADVERTISED_LISTENERS=PLAINTEXT://localhost:9092 \
+    -e KAFKA_CFG_LISTENER_SECURITY_PROTOCOL_MAP=CONTROLLER:PLAINTEXT,PLAINTEXT:PLAINTEXT \
+    -e KAFKA_CFG_CONTROLLER_LISTENER_NAMES=CONTROLLER \
+    -e KAFKA_CFG_INTER_BROKER_LISTENER_NAME=PLAINTEXT \
+    bitnami/kafka:3.6
+}
+
+# Check if the -k parameter is provided
+if [ "$1" == "-k" ]; then
+    start_kafka
+fi
+
+# Set the timeout in seconds
+timeout=180
+# Set the interval between checks in seconds
+interval=5
+
+# Calculate the end time
+end_time=$((SECONDS + timeout))
+
+while [ $SECONDS -lt $end_time ]; do
```

Comment:
please run shellcheck linter, I think it will fail this

Reply:
How can i test these changes locally?

Reply:
by running the script - that was the whole point of the ticket, to enable local integration test